src/style.css

```diff
@@ -0,0 +1,52 @@
+* {
+  margin: 0;
+  padding: 0;
+  --active_star: url(images/star-active.svg);
+}
+
+.stars__block {
+  align-content: center;
+  justify-content: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 140px;
+  height: 160px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  height: 16px;
+}
+
+.stars__star {
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px ;
+}
+
+.stars--5 .stars__star {
+  background-image: var(--active_star);
+}
+
+.stars--4 :not(:last-child) {
+  background-image: var(--active_star);
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: var(--active_star);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: var(--active_star);
+}
+
+.stars--1 :first-child {
+  background-image: var(--active_star);
+}
```

Comment:
Pay attention to the [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md):
2) [CODE STYLE] - If several selectors MUST always have the same styles, group them using , to prevent accidental out ot sync in future